spec/dummy/rails/config/initializers/assets.rb

```diff
@@ -8,4 +8,4 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( all.css all.js )
```

Comment:
Why not just call the default files application.js/css instead of all.js/css? Then we wouldn't need this line.